### PR TITLE
CEDS-2226

### PR DESCRIPTION
**Pull request by alsahid1993**
Updated FE elements within
representative_details.scala
goods_location.scala

Jira: https://jira.tools.tax.service.gov.uk/browse/CEDS-2226

### CODE REVIEW

File: app/views/declaration/declaration_choice.scala.html [hunk 1]
```diff
@@ -91,16 +91,15 @@
                     checked = form("type").value.contains(DeclarationType.OCCASIONAL.toString)
                 ),
                 RadioItem(
-                    divider = Some(messages("declaration.type.or")),
-                    value = Some(RadioOption.dividerId)
+                    divider = Some(messages("declaration.type.or"))
```
Comment (PatrykRudnicki) [inline app/views/declaration/declaration_choice.scala.html]: This change and line 102 has been changed by @SteveSugden  last time. Not sure if this change with removing value is correct

File: app/views/declaration/goods_location.scala.html [hunk 2]
```diff
@@ -54,7 +54,7 @@
         @pageTitle(messages("declaration.goodsLocation.title"))
 
         @exportsInputText(
-        field = form("code"),
+        field = form("dev-goods-location-code"),
```
Comment (PatrykRudnicki) [inline app/views/declaration/goods_location.scala.html]: why this change?

Comment (PatrykRudnicki) [reply app/views/declaration/goods_location.scala.html]: This won't work, because this filed must be the same like form field

File: app/views/declaration/declaration_choice.scala.html (hunk 1 above)
Comment (SteveSugden) [reply app/views/declaration/declaration_choice.scala.html]: No.  This looks like it's reverting my changes of this morning.

File: test/views/declaration/DeclarationChoiceViewSpec.scala [hunk 3]
```diff
@@ -17,26 +17,18 @@
 package views.declaration
 
 import base.Injector
```
Comment (SteveSugden) [inline test/views/declaration/DeclarationChoiceViewSpec.scala]: I don't think there should be any changes to this test either.  Declaration Choice page as already been GDS - its not part of this ticket

File: app/views/declaration/goods_location.scala.html (hunk 2 above)
Comment (alsahid1993) [reply app/views/declaration/goods_location.scala.html]: I was trying to make the the following element consistent with the prototype's:
<input class="govuk-input" id="dev-local-reference-number" name="dev-local-reference-number" type="text" aria-describedby="dev-local-reference-number-hint">

Anu informed me that the tests were failing when the previous elements were listed as:
<input class="govuk-input govuk-!-width-two-thirds" id="code" name="code" type="text" aria-describedby="code-hint">
